src/main.js

```diff
@@ -131,9 +141,13 @@ bot.on("messageCreate", async (message) => {
   }
 
   // if it is a command
-  if (message.content.charAt(0) === PREFIX) {
+  if (message.content.charAt(0) === configManager.config.prefix ?? "$") {
```

Comment:
With no prefix key or an empty string, commands don't work at all. `$err` et. al. don't even produce console output. Perhaps use `||` in these situations to account for empty strings?